litellm/litellm_core_utils/exception_mapping_utils.py

```diff
@@ -797,6 +797,9 @@ def exception_type(  # type: ignore  # noqa: PLR0915
                         message=f"BedrockException: Rate Limit Error - {error_str}",
                         model=model,
                         llm_provider="bedrock",
+                        type="throttling_error",
```

Comment:
wouldn't we just move this `type` to happen in RateLimitError init (inside `exceptions.py` - https://github.com/BerriAI/litellm/blob/fd2d4254bcd01e924ca4dded36ee4714c33734af/litellm/exceptions.py#L277